javascript/index.js

```diff
@@ -14,52 +14,81 @@ const milUniElement = document.getElementById('milUni');
 const splitsElement = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  printSeconds()
+  printMinutes()
 }
 
 function printMinutes() {
-  // ... your code goes here
+  let minutes = chronometer.getMinutes()
+  if (minutes >= 10){
+    secDecElement.innerHTML = Math.floor(minutes / 10)
+    secUniElement.innerText = minutes % 10
+  } else {
+    minDecElement.innerHTML = 0
+    minUniElement.innerText = minutes
+  }
 }
```

Comment:
Hey Harry, good job, but here you can reuse the chronometer functionality (computeTwoDigitNumber), for example like this:
```
 let minutes = chronometer.computeTwoDigitNumber(chronometer.getMinutes());
  minDecElement.innerHTML = minutes[0];
  minUniElement.innerHTML = minutes[1]
```